shared/local-debug.native.js

```diff
@@ -36,7 +36,7 @@ let config: {[key: string]: any} = {
 
 if (__DEV__ && true) {
   config.isDevApplePushToken = true
-  config.clickableVisible = true
+  config.clickableVisible = false
```

Comment:
Personally, :+1: to doing this, but we should discuss it and PR it separately. Please take it out of this changeset.